### PR TITLE
JSON Documentation Initial Commit; Armor Value Fixing

### PR DESCRIPTION
**Pull request by chatterchats**

### CODE REVIEW

**Review by dealloc — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/.idea/
+/pdfSourceJRD.html
```
Comment (dealloc) [inline .gitignore:2]: what are these files?

File: Writerside/topics/Warbonds.md
Comment (dealloc) [inline Writerside/topics/Warbonds.md:1]: need some documentation here still :)

**Review by dealloc — COMMENTED** [review]
looks awesome, my main gripe is that I'd prefer using code blocks in the markdown rather than images of JSON code.

I feel like images should be reserved for things that can't be represented visually.

File: .gitignore (hunk 1 above)
Comment (chatterchats) [reply .gitignore:2]: .idea is generated by Writerside  
pdfSource* and webHelpJRD2-all.zip are the local version that are generated when doing local tests

File: Writerside/topics/Warbonds.md
Comment (chatterchats) [reply Writerside/topics/Warbonds.md:1]: Yee still needs some fleshing out, but I'd already commited the docs that I wasn't planning on doing quite yet, but there was some errors in the armor JSON and some updates I could push in the passive.json and I was only really using master branch so both kinda got included

**Comment by chatterchats** [comment]
https://chatterchats.github.io/json/readme.html -> Current preview link to see how the documentation would look

**Review by dealloc — APPROVED** [review]

**Pull request merged**